src/main/resources/assets/computercraft/lua/rom/programs/which

```diff
@@ -0,0 +1,13 @@
+--Made by Wilma456
```

Comment:
@dan200  What's your view on credit like this? Now that CC is open source should we be adding script author information? I know that treasure disks give their credit on the disk.